integrated_channels/moodle/client.py

```diff
@@ -228,7 +325,27 @@ def delete_content_metadata(self, serialized_data):
 
     @moodle_request_wrapper
     def create_course_completion(self, user_id, payload):  # pylint: disable=unused-argument
-        pass
+        completion_data = json.loads(payload)
+
+        course_id = self.get_course_id(completion_data['courseID'])
+        course_module_id, module_name = self.get_course_final_grade_module(course_id)
+        moodle_user_id = self.get_creds_of_user_in_course(course_id, user_id)
+
+        params = {
+            'wsfunction': 'core_grades_update_grades',
+            'source': module_name,
+            'courseid': course_id,
+            'component': 'mod_assign',
+            'activityid': course_module_id,
+            'itemnumber': 0,
+            'grades[0][studentid]': moodle_user_id,
+            # The grade is exported as a decimal between [0-1]
+            # This could potentially be another point of configuration for the user
+            # as they could potentially supply us with how they want grades displayed (percent, point, letter)
```

Comment:
I assume you mean the grade on edx's side is a decimal (TIL if so) and we can add logic to pass this to Moodle however the customer would like? Can you even do a letter (ie is the field on Moodle's side expecting an int? Or is it just a char field?

Reply:
I added the percent grade to the exportation, and I believe (some investigation pending?) that things like letter grades should be accessible too. And yeah, the main point of the comment was to say that we could allow the customer to configure this and I'm wondering if we want to add the functionality to this PR

Reply:
To add more clarity- how I see this configuration (+ creating shell final grade assignment configurations?), we'd probably add another admin form/model for MoodleCompletionDataConfiguration or something similarly named.

Reply:
Yeah but my question was does Moodle support a letter grade?
I ask because their date field (which is an int [commence groaning]) is strictly typed and I wasn't sure if this grade field is also strictly typed since you are currently passing an int. 

TL;DR if you set grade to "B" here would it fail to load into Moodle?

Reply:
Looking at the UI on Moodle's side, the activities (what the user has to create in order for us to send them grades) don't seem to support letter grades, though there is a _different_ grade "item" (the one I originally thought would be our solution to our implementation) in Moodle that supports a grade type of `text`. It feels like this is how you'd get a letter grade. However having investigated and now discussed with the team, I don't think that configurable options are the best play for this PR. We can communicate with customers going forward and if there's interest then I think we can refer back to this thread.

Reply:
yeah I think we may want to doc this somewhere on what to expect on grading .. or sth like it